sdcm/utils/chaos_mesh.py

```diff
@@ -0,0 +1,62 @@
+import json
+import logging
+
+from sdcm.cluster_k8s import KubernetesCluster
+from sdcm.utils.k8s import HelmValues, KubernetesOps
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ChaosMesh:  # pylint: disable=too-few-public-methods
+    NAMESPACE = "chaos-mesh"
+    VERSION = "2.5.0"
+    HELM_SETTINGS = {'dashboard.create': True, 'dnsServer.create': True}
```

Comment:
It must have `affinity` field with the rule for the Scylla node pools.
Instance of the `KubernetesCluster` has `POOL_LABEL_NAME` and `SCYLLA_POOL_NAME` attributes which must be used in the another K8S utility like in the following example:

```
from sdcm.utils.k8s import get_helm_pool_affinity_values

HELM_SETTINGS = get_helm_pool_affinity_values(k8s_kluster.POOL_LABEL_NAME, k8s_kluster.SCYLLA_POOL_NAME)
HELM_SETTINGS |= {
    'dashboard.create': True,
    'dnsServer.create': True,
}
```

Reply:
I tried setting affinity like this - didn't work, anyway chaos-mesh daemon pods got installed on all the nodes.

I propose to park it in followup task if you agree.

Reply:
ok, nevermind, I made it working - just needed to nest it under `chaosDaemon`.
Thanks @vponomaryov

Reply:
There are also other resources like `controllerManager`, `dnsServer` (if we decide to use dns faults), `dashboard` (but I think we won't be using it). What pool use for them? auxiliary-pool?

Reply:
Yes, `auxiliary-pool` / `default-pool`.